bot/resources/tags/dunder-methods.md

```diff
@@ -1,10 +1,12 @@
-### Dunder methods
+**Dunder methods**
 
 Double-underscore methods, or "dunder" methods, are special methods defined in a class that are invoked implicitly. Like the name suggests, they are prefixed and suffixed with dunders. You've probably already seen some, such as the `__init__` dunder method, also known as the "constructor" of a class, which is implicitly invoked when you instantiate an instance of a class.
 
-When you create a new class, all the default dunder methods are inherited from its superclass (which is `object` if no superclass is specified). However, we can override them by redefining the methods within this new class. For example, the default `__init__` method from `object` doesn't take any arguments, so we almost always override that to fit our needs.
+When you create a new class, all methods - including dunder methods - are inherited from its superclass, which is by default `object`. `object` has a set of default dunder methods, but we can override them by redefining the methods within this new class. For example, the default `__init__` method from `object` doesn't take any arguments, so we almost always override that to fit our needs.
 
-Other common dunder methods to override are `__str__` and `__repr__`. `__str__` is the user-friendly string representation of an object, and is implicitly called on arguments passed into the `str` function. `__repr__` is the developer-friendly string representation of an object - usually the syntax to recreate it - and is called by the `repr` function.
+> NOTE: You can use `dir(object)` to see the available default dunder methods.
```

Comment:
I'm not sure on this. The `NOTE` seems a bit extra. Giving it a moments thought I think we shouldn't include this--one of the goals with tags is as a famous engineering adage goes: 

> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away

The goal here is to explain dunders correctly, succinctly, and with the goal of educating the reader. I think this note could probably be dropped to make this easier to read and digits without losing much of the overall success.

Reply:
got it, i'll delete it